leapp/dialogs/dialog.py

```diff
@@ -34,7 +34,7 @@ def __init__(self, scope, reason, title=None, components=None, key=None):
         :type key: str
         """
         self.components = components or self.components
-        self.title = title
+        self.title = title.replace('\n', ' ').strip() if title else title
```

Comment:
@fernflower  just curious, why the condition? I thought the title is mandatory and cannot be anything else than string. Unless it can be `None` object, why to have the condition?

Reply:
Title can be None, calling replace on None would result in AttributeError

Reply:
@fernflower thx for the explanation. It's weird to me that title can actually `None` at all, but the condition makes sense now.